opentreemap/treemap/lib/map_feature.py

```diff
@@ -22,6 +22,18 @@
 from treemap.util import leaf_subclasses
 
 
+def _photo_upload_share_text(feature, has_tree=False):
+    base = trans("I added a photo of this %s!")
+    if not feature.is_plot:
+        entity = trans("resource")
```

Comment:
I think this should use `MapFeature.display_name`: https://github.com/OpenTreeMap/OTM2/blob/master/opentreemap/treemap/models.py#L626-L629